docs/user-guide/persistent-volumes/index.md

```diff
@@ -222,6 +222,51 @@ A volume will be in one of the following phases:
 
 The CLI will show the name of the PVC bound to the PV.
 
+### Mount Options
+
+Kubernetes administrator can specify additional mount option to be used when a Persistent Volume is
+being mounted on a node. 
+
+Mount option can be specified via an annotation on PV - `volume.beta.kubernetes.io/mount-options`. 
+For example:
+
+```yaml
+apiVersion: "v1"
+kind: "PersistentVolume"
+metadata:
+  name: gce-disk-1
+  annotations:
+    volume.beta.kubernetes.io/mount-options: "discard"
+spec:
+  capacity:
+    storage: "10Gi"
+  accessModes:
+    - "ReadWriteOnce"
+  gcePersistentDisk:
+    fsType: "ext4"
+    pdName: "gce-disk-1
+```
+
+A mount option can be a string which will be cumulatively joined and used while mounting volume to the disk. 
```

Comment:
Can be a string? Or must be a string? What all can it be?

"A mount option is a string which is cumulatively joined and used while mounting the volume to the disk."